.gitignore

```diff
@@ -0,0 +1,6 @@
+MANIFEST
+*.pyc
+.project
+.pydevproject
+.settings
+/python/test/cloudstorage
```

Comment:
I had the folder in there to test the unit test. Didn't think that should be uploaded. Ill upload a cleaner version